.changeset/eighty-bikes-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': patch
+---
+
+fix: Added missing context of `svelte.js` for validation
```

Comment:
We've always started changelog messages with a lowercase letter https://github.com/sveltejs/svelte/blob/main/packages/svelte/CHANGELOG-pre-5.md
```suggestion
fix: added missing context of `svelte.js` for validation
```